docs/UserGuide.md

```diff
@@ -1346,17 +1336,7 @@ Important notes:
 
 ###### Example 2
 
```

Comment:
I can't comment on the line itself but for "example 1", I think it's possible to remove the first screenshot showing the "Opened UniCa$h summary window" and just leave the second window, since the first one doesn't really add any value